lcr/__init__.py

```diff
@@ -48,43 +58,41 @@ def _login(self, user, password):
 
         # Enter the username
         login_input = WebDriverWait(self.driver, TIMEOUT).until(
-                        ec.presence_of_element_located(
-                            (By.CSS_SELECTOR, "input#okta-signin-username")
-                            )
-                        )
+            ec.presence_of_element_located(
+                (By.CSS_SELECTOR, "input#okta-signin-username")
+            )
+        )
         login_input.send_keys(user)
         login_input.submit()
 
         # Enter password
         password_input = WebDriverWait(self.driver, TIMEOUT).until(
-                ec.presence_of_element_located(
-                    (By.CSS_SELECTOR, "input.password-with-toggle")
-                    )
-                )
+            ec.presence_of_element_located(
+                (By.CSS_SELECTOR, "input.password-with-toggle")
+            )
+        )
         password_input.send_keys(password)
         password_input.submit()
 
-        # Wait until the page is loaded
         WebDriverWait(self.driver, TIMEOUT).until(
-                ec.presence_of_element_located(
-                    (By.CSS_SELECTOR, "input#memberLookupMain")
-                    )
-                )
+            ec.presence_of_element_located((By.CSS_SELECTOR, "input[type='search']"))
```

Comment:
This is the current CSS_SELECTOR that we want to wait for to make sure the search bar (and the rest of the page) have loaded before capturing cookies.